src/js/providers/tracks-mixin.js

```diff
@@ -98,7 +98,7 @@ define(['../utils/underscore',
 
         if (this._textTracks) {
             if (this._renderNatively) {
-                this.addTracksListener(this._textTracks, 'change', textTrackChangeHandler.bind(this));
+                this.addTracksListener(this.video.textTracks, 'change', textTrackChangeHandler.bind(this));
```

Comment:
This is somewhat confusing - we keep ```this._textTracks``` and check if that exists, and use ```this.video.textTracks``` to add listener to. It feels like we are unnecessarily keeping two copies of the same thing.